packages/components/src/input/src/Input.css

```diff
@@ -1,32 +1,32 @@
 .o-ui {
```

Comment:
not sure if we care, but clicking on the loading icon makes the input lose focus instead of gain.
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/5cf52469-4e2c-4933-ab7f-87a7b5226e72)

Reply:
If its the same on https://orbit.sharegate.design/ I would not fix this.